fastlane/README.md

```diff
@@ -0,0 +1,54 @@
+fastlane documentation
+================
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
+
+```
+xcode-select --install
+```
+
+Install _fastlane_ using
+```
+[sudo] gem install fastlane -NV
+```
+or alternatively using `brew install fastlane`
+
+# Available Actions
+## Android
+### android test
+```
+fastlane android test
+```
+Runs all the tests
+### android beta
+```
+fastlane android beta
+```
+Submit a new Beta Build to Crashlytics Beta
+### android deploy
+```
+fastlane android deploy
+```
+Deploy a new version to the Google Play
+### android tests
+```
+fastlane android tests
+```
+
+### android upload_screenshots
+```
+fastlane android upload_screenshots
+```
+
+### android capture_screen
+```
+fastlane android capture_screen
+```
+
+
+----
+
+This README.md is auto-generated and will be re-generated every time [fastlane](https://fastlane.tools) is run.
```

Comment:
Please, let's not commit auto-generated files

Reply:
Added to gitignore

Reply:
is there a way to remove it?

Reply:
git rm -rf fastlane/README.md